src/main/java/com/documaster/rms/noark/ws/constants/Kassasjonshjemmel.java

```diff
@@ -0,0 +1,26 @@
+package com.documaster.rms.noark.ws.constants;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Kassasjonshjemmel extends NoarkEnum {
```

Comment:
Should this list value and the other new one be added to `NoarkObjectMappings#createCodeValueMappings`?

Reply:
Added with https://github.com/documaster/ws-client-n5-java/pull/27/commits/84b7aaaa9dac2d892c8b473a1c377dc9ef5ee6ab